test/util.js

```diff
@@ -103,6 +103,15 @@ function getTokens(options = {}) {
   });
 }
 
+function getKeySet() {
+  return fetch(ISSUER + '/v1/keys')
+  .then(response => {
+    if (response.ok) {
+      return response.json();
+    }
```

Comment:
Should we handle when `!response.ok`? Since it won't throw if the request completed successfully but is non 2xx